Buildings/Applications/DHC/CentralPlants/Cooling/Controls/ChilledWaterPumpSpeed.mo

```diff
@@ -0,0 +1,113 @@
+within Buildings.Applications.DHC.CentralPlants.Cooling.Controls;
+model ChilledWaterPumpSpeed
+  "Controller for up to two headed variable speed chilled water pumps"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Integer numPum(final min=1, final max=2)=2
+    "Number of chilled water pumps, maximum is 2";
+  parameter Modelica.SIunits.PressureDifference dpSetPoi(displayUnit="Pa")
+    "Pressure difference setpoint";
+  parameter Modelica.SIunits.Time tWai "Waiting time";
+  parameter Modelica.SIunits.MassFlowRate m_flow_nominal
+    "Nominal mass flow rate of single chilled water pump";
+  parameter Real minSpe(final unit="1",final min=0,final max=1) = 0.05
+    "Minimum speed ratio required by chilled water pumps";
+  parameter Modelica.SIunits.MassFlowRate criPoiFlo = 0.7*m_flow_nominal
+    "Critcal point of flowrate for switching pump on or off";
+  parameter Modelica.SIunits.MassFlowRate deaBanFlo = 0.1*m_flow_nominal
+    "Deadband for critical point of flowrate";
+  parameter Real criPoiSpe = 0.5
+    "Critical point of speed signal for switching on or off";
+  parameter Real deaBanSpe = 0.3
+    "Deadband for critical point of speed signal";
+  parameter Modelica.Blocks.Types.SimpleController controllerType=
+    Modelica.Blocks.Types.SimpleController.PI
+    "Type of pump speed controller";
+  parameter Real k(final unit="1", final min=0) = 1 "Gain of controller";
+  parameter Modelica.SIunits.Time Ti(final min=Modelica.Constants.small)=60
+    "Time constant of Integrator block" annotation (Dialog(enable=
```

Comment:
refactor the text to have better format. Also, better to group the parameter `k`, `Ti`, `Td` in a group like `Speed controller`.